db/seeds.rb

```diff
@@ -43,6 +43,21 @@ class SeedAdminENVError < StandardError
   end
 end
 
+# This populates the 'regions' table for Swedish regions (aka counties),
+# as well as 'Sweden' and 'Online'.  This is used to specify the primary
+# region in which a company operates.
+#
+# This uses the 'city-state' gem for a list of regions (name and ISO code).
+# (That gem will also return a list of cities within region)
+#
+# If the regions table is not empty, do not run this code unless there
+# are also no companies, as the region ID's will be reset.
+if Region.exists?
```

Comment:
Will this situation be an issue if we're seeding the entire db? 
If so, then can we have a way to load the region table without clobbering existing region information?
(I'm not sure this will be an issue if we're seeding the entire db.)